extensions/amp-story/1.0/test/test-amp-story-store-service.js

```diff
@@ -20,7 +20,7 @@ import {
   StateProperty,
   UIType,
 } from '../amp-story-store-service';
-import {EmbedMode, EmbedModeParam} from '../embed-mode';
+import {EmbedMode, EmbedModeParam} from '../../../../src/amp-story-embed-mode';
```

Comment:
Why are we moving this file? It clearly belongs in the `/amp-story` folder. We don't even have OWNERS rights on the `/src` folder.

The Player is one client for AMP Story and should do like other clients that have to duplicate the EmbedMode enum. I'm fine with either duplication or importing the enum from `/amp-story` (and whitelisting the import so the compiler doesn't complain), but moving the file is confusing.